csrankings-m.csv

```diff
@@ -1475,6 +1475,7 @@ Mengjia Yan 0001,Massachusetts Institute of Technology,https://people.csail.mit.
 Mengjie Zhang 0001,Victoria University of Wellington,http://ecs.victoria.ac.nz/Main/MengjieZhang,hLvGrrkAAAAJ
 Mengjuan Liu,UESTC,https://www.is.uestc.edu.cn/teachers.do?id=1085,NOSCHOLARPAGE
 Mengmeng Ge,RMIT University,https://www.rmit.edu.au/contact/staff-contacts/academic-staff/g/ge-dr-mengmeng,dj2SBEIAAAAJ
+Mengnan Du,NJIT,https://mengnandu.com/,8uXBeaEAAAAJ

```

Comment:
Need to show how you meet the inclusion criteria (ability to solely advise a PhD student for a PhD _in Computer Science_).

Reply:
Hi Dr. Berger,

Thanks for the message. I have a joint appointment in CS. Currently I am
advising two CS PhD students as the sole advisor. My position at NJIT is 100% time. 

Best,
Mengnan